Source/WebCore/css/parser/CSSPropertyParserConsumer+Anchor.cpp

```diff
@@ -306,8 +306,11 @@ RefPtr<CSSValue> consumePositionArea(CSSParserTokenRange& range, const CSSParser
         return CSSPrimitiveValue::create(CSSValueNone);
 
     auto maybeDim2 = consumeIdentRaw(range);
-    if (!maybeDim2)
+    if (!maybeDim2) {
+        if (!getKeywordType(dim1))
```

Comment:
This is rather embarrassing 😅 the parsing test for `position-area` doesn't test this case.